app/components/auth/signUp/components/surveyForm/components/question.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { Dispatch } from 'redux';
+import { Survey, AnswerToQuestion } from '../constants';
+import { withStyles } from '../../../../../../helpers/withStylesHelper';
+import { ActionCreators } from '../../../../../../actions/actionTypes';
+const styles = require('./question.scss');
+
+interface QuestionProps {
+  question: Survey;
+  qKey: number;
+  dispatch: Dispatch<any>;
+}
+
+interface AnswerProps {
+  value: string;
+  name: string;
+  type: string;
+  handleOnClickAnswerToSurvey: () => void;
+}
+
+function onClickAnswerToSurvey(survey: AnswerToQuestion, type: string, dispatch: Dispatch<any>) {
+  dispatch(ActionCreators.clickToAnswerInSurveyForm({ survey, type }));
+}
+
+const Answer: React.FC<AnswerProps> = React.memo(props => {
```

Comment:
In this case, memo doesn't work.
because `handleOnClickAnswerToSurvey` prop is remade every time where the parent component is re-rendered.